.github/workflows/pr.yml

```diff
@@ -386,11 +386,20 @@ jobs:
   verify-autoconsent-bundle:
     name: 'Verify autoconsent bundle'
     runs-on: ubuntu-latest
-
     steps:
-      - name: Skip
+      - uses: actions/checkout@v4
```

Comment:
Are these changes intentional? Where are they coming from? Should revert?

Reply:
yep, it's basically reverting the earlier change from [here](https://github.com/duckduckgo/macos-browser/pull/3086#discussion_r1713969651)